azkustoingest/common.go

```diff
@@ -0,0 +1,109 @@
+package azkustoingest
```

Comment:
Why did you choose the name "Common"?
Isn't Options more suitable

Reply:
renamed to ingestion_options